codegen/templates/grpc_client.tmpl

```diff
@@ -75,6 +75,7 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 		{{ end -}}
 		opts: zanzibar.NewGRPCClientOpts(
 		deps.Default.Logger,
+		deps.Default.ContextLogger,
```

Comment:
can we not just replace the deps.Default.Logger with ContextLogger instead of adding both of them.